### PR TITLE
Add Experiment Details Metric Assignments Panel

### PR DESCRIPTION
**Pull request by dhurlburtusa**
Add Experiment details Metric Assignments panel.

## How has this been tested?

New tests for new component.
Automated tests still pass with 100% coverage.

### CODE REVIEW

File: components/ExperimentDetails.tsx [hunk 1]
```diff
@@ -3,18 +3,32 @@ import React from 'react'
...
+  segments: Segment[]
+}) {
   debug('ExperimentDetails#render')
   return (
     <div>
       <h2>Experiment details</h2>
       <GeneralPanel experiment={experiment} />
       <br />
       <AudiencePanel experiment={experiment} segments={segments} />
+      <br />
```
Comment (dhurlburtusa) [inline components/ExperimentDetails.tsx:30]: Responsive layout of the panels will come in a later PR.

**Comment by yanirs** [comment]
@dhurlburtusa looks like this PR includes #166. Can you please update it to be standalone?

File: components/MetricAssignmentsPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    const metric = metricsById[metricAssignment.metricId]
+
+    if (!metric) {
+      throw Error(
+        `Failed to lookup metric with ID ${metricAssignment.metricId} for assignment with ID ${metricAssignment.metricAssignmentId}.`,
+      )
+    }
+
+    return {
+      attributionWindowSeconds: metricAssignment.attributionWindowSeconds,
```
Comment (yanirs) [inline components/MetricAssignmentsPanel.tsx:39]: Can we replace these with `...metricAssignment`?

Comment (dhurlburtusa) [reply components/MetricAssignmentsPanel.tsx]: Yep. Done.

**Pull request merged**